### PR TITLE
Document precompilation limitations of global variables

### LINKED ISSUE #965: Document __init__ workaround for global variables
If one wants to use global variables involving Nemo objects that are backed by pointer shenanigans (so almost any type), one has to do it using `__init__`. This comes up regularly (see nemo-devel mailing list). We should find a place in the documentation to write this down.

### PR DESCRIPTION
**Pull request by thofma**
Closes #965. Please check my English :)

### CODE REVIEW

File: docs/src/misc.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Miscellaneous
+
+## Global variables and precompilation
+
+Due to limitations of the precompilation of modules julia, global variables
```
Comment (wbhart) [inline docs/src/misc.md:5]: in julia?

**Comment by wbhart** [comment]
It's perfect. Just one possible tpyo.

File: docs/src/misc.md (hunk 1 above)
Comment (thofma) [inline docs/src/misc.md:5]:
```suggestion
Due to limitations of the precompilation of modules in julia, global variables
```

**Comment by thofma** [comment]
Thansk!

**Pull request merged**